### ISSUE TITLE
fails to parse list of INPUT files

### ISSUE THREAD

**Issue by simevo** [doxygen/doxygen#8297]
**Describe the bug**
doxygen 1.9.0 fails to build the html documentation of [starpu](http://starpu.gforge.inria.fr/), it runs for 0.2 seconds and the resulting html dir is 0.4 MB; no error is issued ...

while parsing the first meaningful line of the config file:
```
INPUT                  = ../../../doc/doxygen/chapters \
```
it reports the following warnings:
```
warning: ignoring unknown character '.' at line 14, file doxygen-config.cfg
warning: ignoring unknown character '.' at line 14, file doxygen-config.cfg
warning: ignoring unknown character '/' at line 14, file doxygen-config.cfg
warning: ignoring unknown character '.' at line 14, file doxygen-config.cfg
warning: ignoring unknown character '.' at line 14, file doxygen-config.cfg
warning: ignoring unknown character '/' at line 14, file doxygen-config.cfg
warning: ignoring unknown character '.' at line 14, file doxygen-config.cfg
warning: ignoring unknown character '.' at line 14, file doxygen-config.cfg
warning: ignoring unknown character '/' at line 14, file doxygen-config.cfg
warning: ignoring unknown tag 'doc' at line 14, file doxygen-config.cfg
warning: ignoring unknown character '/' at line 14, file doxygen-config.cfg
warning: ignoring unknown tag 'doxygen' at line 14, file doxygen-config.cfg
warning: ignoring unknown character '/' at line 14, file doxygen-config.cfg
warning: ignoring unknown tag 'chapters' at line 14, file doxygen-config.cfg
warning: ignoring unknown character '/' at line 14, file doxygen-config.cfg
warning: ignoring unknown tag 'api' at line 14, file doxygen-config.cfg
```

**Expected behavior**
doxygen 1.8.20 runs for 2 seconds and the resulting html dir is 11 MB

**Screenshots**

**To Reproduce**

**Version**
1.9.0 from Debian experimental

**Stack trace**

**Additional context**

**Comment by albert-github** [comment]
Nice testing.

Missing here the build instructions, though was able to reproduce the problem with:
- download starpu-1.3.7.tar.gz form the mentioned site (i.e. https://files.inria.fr/starpu/starpu-1.3.7/starpu-1.3.7.tar.gz)
- cd starpu-1.3.7/doc/doxygen
- copy `doxygen-config.cfg.in` to `doxygen-config.cfg`
- replace `@top_srcdir@` and `@top_builddir@` with `../..` in `doxygen-config.cfg`
- comment out the line `INPUT_FILTER           = ../doc/doxygen/doxygen_filter.sh` (not relevant here).
- run doxygen `doxygen doxygen.cfg`

First analyses give that even a Doxyfile like:
```
#
INPUT                  = \
                         ../../include/schedulers/starpu_heteroprio.h
QUIET=YES
```
give the error (looks like the # line together with the continuation (`\`) is required).

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #8298

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by simevo** [comment]
The patch fixes the issue with starpu. Thanks !

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.1.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
